### ISSUE TITLE
Menu Items

### ISSUE THREAD

**Issue by Umbyology** [getgrav/grav-theme-hpstr#24]
So, as I create new pages, post etc ... new menu items are NOT automatically created?

**Comment by ricardo118** [comment]
yeah this is a very old theme, hasnt been updated to automatically add navigation. it is set in the site.yaml config file
